### ISSUE TITLE
0.4.1 compile error (No such var: tagged/message-out)

### ISSUE THREAD

**Issue by keeth** [nervous-systems/fink-nottle#6]
When compiling a project that includes 0.4.1 I get:
java.lang.RuntimeException: No such var: tagged/message-out, compiling: (fink_nottle/internal/sqs.cljc:45:18)

```
other:guru-etl [master] keith $ jar -tf /Users/keith/.m2//repository/io/nervous/fink-nottle/0.4.1/fink-nottle-0.4.1.jar |grep tagged
fink_nottle/sqs/tagged.cljc
fink_nottle/sqs/tagged.clj
```

the file tagged.clj contains only a namespace decl.  it appears to be clobbering tagged.cljc.

my deps:

```
  :dependencies [[org.clojure/clojure "1.7.0"]
                 [org.clojure/core.async     "0.1.346.0-17112a-alpha"]
                 [org.clojure/tools.logging  "0.3.1"]
                 [io.nervous/fink-nottle "0.2.0"]
                 [http-kit      "2.1.19"]
                 [cheshire      "5.5.0"]]
```

**Comment by moea** [comment]
That's what I get for pushing a release from a dirty working copy.  I just released 0.4.2 - any luck?

**Comment by keeth** [comment]
yep, fixed. thanks for the quick response!
